spec/unit/type/gluster_peer_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
Rabbitmq module seems to have tests under `spec/unit/puppet/type`? I guess it's not important...